### PR TITLE
Add content 5

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no "About" section. #4 
- Currently there is no "Impressum" / About page. #2 

**What is the new behavior (if this is a feature change)?**
- An about section has been added. #4 
- An "Impressum" / About page has been added. #2 

**Other information**:
- The version has been updated to v1.0.0 (ready for launch)

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
Looks ok to me

**Review by schettn — APPROVED** [review]
Should be ok to merge.

File: src/components/organisms/About/index.jsx [hunk 1]
```diff
@@ -33,8 +30,70 @@ class About extends React.Component {
...
+            Das ist die Tragik und die Schönheit der Natur. Ein ewiger Kreislauf. Alles muss irgendwann enden, 
+            um neues zu schaffen. Diese Ideologie spiegelt sich in meinen Werken wieder. Ich arbeite mit der 
+            Natur - mit dieser Idee - ich arbeite mit mir.
+            </p>
+            <p>
+            Mit jedem Bild sterbe ich.<br/>
+            Mit jedem Bild werde ich neu geboren.
+            </p>
+            <p>
+            Gedanken, Gefühl und Erinnerungen gehen und neues Wissen und Verständnis kommt - das ist meine
```
Comment (sweggerat) [inline src/components/organisms/About/index.jsx:64]: , expected after "gehen"

**Review by sweggerat — APPROVED** [review]
One , expected.

**Pull request merged**